### DISCUSSION TITLE
QT6 support

### DISCUSSION THREAD

**Discussion by miketoon** [collin80/SavvyCAN#551] [General]
Do you have plans to support QT6? QT5.15 is no longer available through the QT Maintenance Tool so it is difficult to build locally without this. I noticed from this thread that the blocker was absence of the SerialBus library, but this appears to be available now:
https://savvycan.com/index.php/forum/welcome-mat/234-savvycan-qt-install-help

Mike

**Comment by miketoon** [comment]
Not for me on a recent install:
![image](https://user-images.githubusercontent.com/32300092/205935039-21aee650-a1c9-4555-af91-0f8e3c66a0e7.png)

**Comment by collin80** [comment]
Hmm, I'm not sure what's going on. It was my understanding that 5.15.2 should be available for the foreseeable future. But, to answer the question - yes. But, I don't have a time table for when it would be done. There have been some commits to bring the code closer to QT6 support but it is not finished and currently does not compile properly under QT6.
